cva6/tests/custom/sv32/vm_pmp_check_01.S

```diff
@@ -0,0 +1,203 @@
+###################################################################################################
+# Verification Goal:   Remove X PMP permission of the code section and R/W permission of the data #
```

Comment:
why the description of all these tests seem so off. while the description of this test cva6/tests/custom/sv32/vm_rwx_access_smode_01.S looks good

Reply:
@YazanHussnain-10x

Reply:
Done